src/location-registry/controllers/controller.py

```diff
@@ -1,13 +1,16 @@
 from flask import Blueprint, request, jsonify
```

Comment:
Since the file is already in the controllers directory, could we use a **filename** that is in line with our Sequence Diagrams?
Can we use a verb statement? More like: **create-site**? or **create-location**?
I would go with `site` instead of `location` in all our future refactoring/renaming....